Makefile

```diff
@@ -98,22 +98,29 @@ test: manifests generate fmt vet envtest ## Run tests.
 build: generate fmt vet ## Build manager binary.
 	go build -o bin/manager main.go
 
-build-vgmanager: generate fmt vet ## Build manager binary.
+build-vgmanager: generate fmt vet ## Build vg manager binary.
 	go build -o bin/vgmanager cmd/vgmanager/main.go
 
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
-docker-build: test ## Build docker image with the manager.
```

Comment:
Can we leave this in? It is a good way to make sure everything passes.

Reply:
- As part of CI we are caching `go mod` and running `test` as standalone
- if docker-build also triggers `test` it's futile and only lengthens time